namespaces/cloud-platform-live-0.k8s.integration.dsd.io/kube-ops/oidc-proxy-secret.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
~This is likely not needed, we're not using `oidc-proxy` for this?~

Can you please rename the file to something more accurate (eg.: `oidc-credentials.yaml`) and also remove the `cookie-secret` value and comment (AFAIK it's not used?)